ecl/network/v2/load_balancer_interfaces/requests.go

```diff
@@ -61,19 +61,19 @@ type UpdateOpts struct {
 	Description *string `json:"description,omitempty"`
 
 	// IP Address
-	IPAddress string `json:"ip_address,omitempty"`
+	IPAddress *interface{} `json:"ip_address,omitempty"`
```

Comment:
*string looks good, but why is it *interface{}?

Reply:
Thank you for your advice! I fixed. IPAddress does not have to be `*interface{}`.

Reply:
understood. Thank you.